File: Difficulty: Easy/Bubble Sort/bubble-sort.py
class Solution:
    def bubbleSort(self,arr):
        # code here
        n = len(arr)
        
        for i in range(n):
            for j in range(0,n-1-i) : 
                if arr[j] > arr[j+1] :
                    temp = arr[j]
                    arr[j] = arr[j+1]
                    arr[j+1] = temp
                    
        return arr

File: Difficulty: Easy/Bubble Sort/bubble-sort.cpp
class Solution {
  public:
    void bubbleSort(vector<int>& arr) {
        // code here
        int n = arr.size();
        
        for(int i=0; i<n-1 ; i++) {
            bool swapped = false;
            
            for(int j=0; j<n-1-i ; j++) {
                if(arr[j] > arr[j+1]) {
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                    swapped = true;
                }
            }
            if(!swapped) break;
        }
        return;
    }
};
